src/main/resources/application-test.yml

```
@@ -26,4 +26,12 @@ application:
   gemini:
-    api-key: sdlkfnsdlk
\ No newline at end of file
+    api-key: sdlkfnsdlk
+  nomic:
+    api-key: asdfasdsdf
+  pgvector:
+    host: asdas
+    port: 2313
+    db: sdfdfgv
+    user: as3ds
+    password: r43rfsd
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)